src/d/actor/d_a_obj_swpush.cpp

```diff
@@ -5,212 +5,802 @@
+        /* mSpeedDecay      */ 0.6f,
+        /* mPushSpeed0      */ 0.0f,
+        /* m34              */ -0.3f,
+        /* m38              */ 1,
+        /* mMiniPushDelay1  */ 6,
+        /* mPushDelay       */ 6,
+        /* mMiniPushDelay2  */ 13,
+        /* m40              */ 1.0f,
+        /* m44              */ 2.5f,
+        /* mPauseDuration   */ 10
+    },
+    {
+        /* mHeapSize        */ 0xAE0,
+        /* mFlags           */ static_cast<AttrFlag_e>(FLAG_UNK20),
+        /* mScale           */ 1.0f,
+        /* mKbotaResName    */ daObjSwpush::Act_c::M_arcname_kbota,
+        /* mHhbotResName    */ NULL,
+        /* mBgArcName       */ daObjSwpush::Act_c::M_arcname_kbota,
+        /* mModelArcName    */ daObjSwpush::Act_c::M_arcname_kbota,
+        /* mBtpArcName      */ NULL,
+        /* mBgResIndex      */ 7,
+        /* mModelResIndices */ { 4, 4 },
+        /* mBtpResIndex     */ 0xFFFF,
+        /* mSpring          */ 0.9f,
+        /* mSpeedDecay      */ 0.6f,
+        /* mPushSpeed0      */ 0.0f,
+        /* m34              */ -0.3f,
+        /* m38              */ 1,
+        /* mMiniPushDelay1  */ 6,
+        /* mPushDelay       */ 6,
+        /* mMiniPushDelay2  */ 13,
+        /* m40              */ 1.0f,
+        /* m44              */ 2.5f,
+        /* mPauseDuration   */ 10
+    },
+    {
+        /* mHeapSize        */ 0xAE0,
+        /* mFlags           */ static_cast<AttrFlag_e>(FLAG_ON_IS_UP | FLAG_STAY_PRESSED | FLAG_OBEY_SAVE),
+        /* mScale           */ 1.0f,
+        /* mKbotaResName    */ daObjSwpush::Act_c::M_arcname_kbota,
+        /* mHhbotResName    */ NULL,
+        /* mBgArcName       */ daObjSwpush::Act_c::M_arcname_kbota,
+        /* mModelArcName    */ daObjSwpush::Act_c::M_arcname_kbota,
+        /* mBtpArcName      */ NULL,
+        /* mBgResIndex      */ 7,
+        /* mModelResIndices */ { 4, 4 },
+        /* mBtpResIndex     */ 0xFFFF,
+        /* mSpring          */ 0.9f,
+        /* mSpeedDecay      */ 0.6f,
+        /* mPushSpeed0      */ 0.0f,
+        /* m34              */ -0.3f,
+        /* m38              */ 1,
+        /* mMiniPushDelay1  */ 6,
+        /* mPushDelay       */ 6,
+        /* mMiniPushDelay2  */ 13,
+        /* m40              */ 1.0f,
+        /* m44              */ 2.5f,
+        /* mPauseDuration   */ 10
+    },
+    {
+        /* mHeapSize        */ 0x8000,
+        /* mFlags           */ static_cast<AttrFlag_e>(FLAG_REQ_HEAVY | FLAG_STAY_PRESSED | FLAG_OBEY_SAVE),
+        /* mScale           */ 1.5f,
+        /* mKbotaResName    */ daObjSwpush::Act_c::M_arcname_kbota,
+        /* mHhbotResName    */ daObjSwpush::Act_c::M_arcname_hhbot,
+        /* mBgArcName       */ daObjSwpush::Act_c::M_arcname_kbota,
+        /* mModelArcName    */ daObjSwpush::Act_c::M_arcname_hhbot,
+        /* mBtpArcName      */ NULL,
+        /* mBgResIndex      */ 7,
+        /* mModelResIndices */ { 4, 4 },
+        /* mBtpResIndex     */ 0xFFFF,
```

Comment:
```suggestion
        /* mBgResIndex      */ KBOTA_00_DZB_KBOTA_00,
        /* mModelResIndices */ { HHBOT_BDL_HHBOT1, HHBOT_BDL_HHBOT1 },
        /* mBtpResIndex     */ -1,
```
You can include a couple of the headers in the d/res folder to get enum names for all the file indexes in these structs.
Also, mBtpResIndex is signed, so -1 makes more sense than 0xFFFF here.